new/openstack-manager-blueprint-inputs.yaml

```diff
@@ -57,9 +57,18 @@
 #############################
 # Security Settings
 #############################
-# Username and password to be used for login, if the simple userstore implementation is used.
-#username: 'admin'
-#password: 'admin'
+# Cloudify REST security is disabled by default. To enable security, set to true.
+#security_enabled: false
```

Comment:
we should probably also state above that if security is not enabled, the other inputs are irrelevant..